### DISCUSSION TITLE
A repository that is not mine is showing up in my list of repositories

### DISCUSSION THREAD

**Discussion by cu** [community/community#66248] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I am using [Pygithub](https://github.com/PyGithub/PyGithub) and a fine-grained PAT to access my account. It's all working fine.

...Except that when I list all of the repositories via the API, one of them doesn't belong to me. This repo does not show up in my repo list in the web UI, and I am not part of any organization or group. When I visit the repo in question, I do not see any way to interact with it beyond what I normally would on someone else's public repo.

For reference, the repo in question is https://github.com/scroobyuk/CodingHTMLforAutoTrading

How can I remove this repo from my list of repositories when fetched from the API?

**Comment by dkt147** [comment]
Maybe it's because maybe you are a collaborator in someone's repo. Please check your email you must had received an invitation email.

**Reply by cu** [reply]
There is nothing in my email, but I also don't keep any email for more than a few weeks. Is there another way to tell whether I am a collaborator of the repo?

If I am, is there a way to stop being a collaborator?

(Again, I don't _seem_ to have any additional rights on this repo, that I can tell from the web UI.)

**Comment by cu** [comment]
I found it: https://github.com/settings/repositories

Good job hiding this, GH.
